client/src/components/graph/graph.tsx

```diff
@@ -635,19 +631,23 @@ class Graph extends React.Component<GraphProps, GraphState> {
       crossfilter,
       pointDilation,
       viewport,
-      spatial,
       imageUnderlay,
       screenCap,
     } = props.watchProps;
     const { modelTF } = this.state;
-    const [layoutDf, colorDf, pointDilationDf] = await this.fetchData(
+    const [layoutDf, colorDf, pointDilationDf, spatial] = await this.fetchData(
       annoMatrix,
       layoutChoice,
       colorsProp,
       pointDilation
     );
     const { currentDimNames } = layoutChoice;
 
+    // This  accessor will be cleaned up post schema migration
```

Comment:
is there a ticket created to clean this up? Let's add a TODO and link the ticket here thank you!